### ISSUE TITLE
New Pezbase Website

### ISSUE THREAD

**Issue by Beakerboy** [Beakerboy/PezBase#1]
@Hardaker587 Lets use this as the home for PezBase code

**Comment by gavin-hardaker** [comment]
> @Hardaker587 Lets use this as the home for PezBase code

Happy to ;)

**Comment by Beakerboy** [comment]
Do you have a development environment? I wrote a couple migration scripts to pull in users, nodes, and taxonomy. I plan on installing Drupal 9.2 to test that out today. Please use this issues queue to list things you need from me.

**Comment by gavin-hardaker** [comment]
Yeah I use Mamp on both my machines for PHP development

**Comment by gavin-hardaker** [comment]
I think the biggest thing i would need right now, after you're done, is a sample data set so I can work with structuring the UI :)

**Comment by Beakerboy** [comment]
I just created a new module called pezbase_examples which will supply some default data. I’ll let you k ow when it is ready. I’m working on the data models right now. I have the taxonomies laid out, and data import scripts for those. I’m working on the stems and heads next. However, I’m guessing you could get the main page layout, menu look, and block theming even without the data.

**Comment by gavin-hardaker** [comment]
Correct! As long I even just know what the basic structure of the data is gonna be I can work with that because them I can plan for null references, or empty images for instance.

**Comment by Beakerboy** [comment]
As a user, let me know if you have any ideas on ways the data could be better presented. In Drupal-land, anything that is a Taxonomy or Node will automatically have some sort of page to present the entity. I’m planning on making the families taxonomies, so we can browse to the items within the family structures. Stems and Heads will obviously be Nodes. I think I’ll make patent numbers as taxonomy terms, because they deserve a small article explaining the date ranges and maybe even links to the patent itself.

 The existing website has the country / imc pairs set up like a single level taxonomy. I think I’ll just make these as a field associated with the Stem Nodes, less overhead and no need to have pages for them.

Do you have an opinion on stem colors? They do not need unique pages, but there are so many that maybe the existing taxonomy organizing interface will help keep them alphabetical and organized.

**Comment by Beakerboy** [comment]
@Hardaker587 Things should be in a state where you can begin working on a theme. In fact, I have begun translating the existing pezbase Theme to drupal just so I can understand the process. I am not a graphic designer so I am sure this translation will not look good, but I should at least know how the pieces work together when I’m done in case you have any questions.
